src/api/mappers/producerMapper.ts

```diff
@@ -1,37 +1,37 @@
+import { Language } from '../../models/language';
 import { Producer, ProducerRelated } from '../../models/producer';
+import { ProducerRelationType } from '../../models/producerRelationType';
+import { ProducerType } from '../../models/producerType';
 import { ProducerDto, ProducerRelatedDto } from '../dtos/producerDto';
 import { ExternalLinksMapper } from './externalLinksMapper';
 
-/**
- * Maps related producer from dto.
- * @param dto Dto.
- */
-function mapRelatedProducerFromDto(dto: ProducerRelatedDto): ProducerRelated {
-  return {
+export namespace ProducerMapper {
+
+  /**
+   * Maps dto into model.
+   * @param dto Dto.
+   */
+  const producerRelatedFromDto = (dto: ProducerRelatedDto): ProducerRelated => ({
```

Comment:
You should have chosen incoming changes over your current on merge with `develop` 😭 I slightly improved docs, replace with `arrow functions` with `function declaration` and move out of `namespace` non-export things.

Reply:
:( I'll fix it